rules/combo/backdoor/php.yara

```diff
@@ -277,3 +277,20 @@ rule php_str_replace_obfuscation : critical {
 	condition:
 		filesize < 65535 and 2 of ($f*) and any of ($i*) and 2 of ($o*)
 }
+
+rule php_cgi_argument_injection : critical {
+  meta:
+    date = "2024-06-06"
+    description = "detect php CGI argument injections"
+    discovered_by = "Orange Tsai (@orange_8361) of DEVCORE (@d3vc0r3)"
+    exploiters = "Aliz (@AlizTheHax0r) and Sina Kheirkhah (@SinSinology) of watchTowr (@watchtowrcyber)"
+    reference = "https://devco.re/blog/2024/06/06/security-alert-cve-2024-4577-php-cgi-argument-injection-vulnerability-en"
+    repository = "https://github.com/watchtowrlabs/CVE-2024-4577"
+    technical = "https://labs.watchtowr.com/no-way-php-strikes-again-cve-2024-4577"
+
+  strings:
+    $url_pattern = /\?%ADd\+allow_url_include%3d1\+(%ADd|-)[d+]\+auto_prepend_file%3dphp:\/\/input/ nocase
```

Comment:
- I think you can leave out `nocase` here. See https://github.com/Neo23x0/YARA-Performance-Guidelines/ for why it should be avoided when possible.
- Removing the leading `\?%ADd\+` portion may yield more hits without adding false positives

Reply:
Removed `nocase` in [`f17f84d` (#266)](https://github.com/chainguard-dev/bincapz/pull/266/commits/f17f84d67fb09ae9e294b6b4fa020cc9c4cc5240), though 👍🏻